File: Algorithm/baekjoon/bj_11399.py
# 그리디 정렬 | bj 11399 ATM
# github.com/chj3748
import sys

def input():
    return sys.stdin.readline().rstrip()


N = int(input())
times = list(map(int, input().split()))
times.sort()
total = 0
for t in times:
    total += N * t
    N -= 1

print(total)

File: Algorithm/baekjoon/bj_11399.java
// 그리디 정렬 | bj 11399 ATM
// github.com/chj3748
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        int n = Integer.parseInt(br.readLine());
        StringTokenizer strings = new StringTokenizer(br.readLine());
        Integer [] numbers = new Integer[n];
        while (strings.hasMoreTokens()) {
            numbers[--n] = Integer.parseInt(strings.nextToken());
        }
        Arrays.sort(numbers, Collections.reverseOrder());
        int total = 0;
        for (int num: numbers) {
            total += num * ++n;
        }
        System.out.println(total);
    }
}
